### ISSUE TITLE
Code for weather and explanation

### ISSUE THREAD

**Issue by ghost** [migueravila/Bento#49]
Hi, I think i found the code for the weather. But i dont know what does it mean? Can anyone explain to me? Thanks

**Comment by David-Lazaro-Fernandez** [comment]
## Hi 🖖👽! @bryantTheCoder 
### Hope you're having an amazing day! 🌞

The documentation explains most of how to use the configuration file, but don't worry!  😄

For setting up the Weather Api, you'll need 5 things 🔽

- [ ] **An Api key** from [https://openweathermap.org/](https://openweathermap.org/)
- [ ] **Your latitude** ↕ 
- [ ] **Your longitude** ↔
> You can get them here 👉 [https://www.latlong.net/](https://www.latlong.net/)
- [ ] **Choose your favorite set of icons** ☀

![image](https://user-images.githubusercontent.com/57787993/135790997-2c421f0e-0447-44ce-97d3-1e030a321a2d.png)

<div align="center">
<b>[Nord] - [One Dark] - [Dark] - [White]</b>
</div>

---

- [ ] **Choose the temp unit system of your preference**    ```Kelvin, Celsius, Farenheid``` ❄☀


So basically you can config the weather by this way 👇

```// Weather
  weatherKey: '//Your Api key from openwathermap',
  weatherIcons: '//You favorite set of icons (Nord, One Dark, Dark, White)',
  weatherUnit: '//Your favorite temperature unit system (C, K, F)',
  weatherLatitude: '//Your latitude',
  weatherLongitude: '//Your longitude',
  ```

**Comment by bryantTheCoder** [comment]
There is config.js and weather.js . But why config.js when there is weather.js?

**Comment by migueravila** [comment]
Hi @bryantTheCoder. `config.js` is for the user, so when you fork the project. You only need to modify that file and the `app.css` file for appearance. `weather.js` is the code that calls the API so we have the information for the app. 

You can see it as `config.js` -> `weather.js` so it can give you the weather.
